### PR TITLE
Updated to use centralised CI and standard format for changelog

### PR DESCRIPTION
**Pull request by alourie**
Signed-off-by: Alex Lourie <alex.lourie@zepben.com>

### CODE REVIEW

**Comment by ryanjcoleman** [comment]
Task linked: [OPS-158 Standardise changelog format across all repos](https://app.clickup.com/t/6929263/OPS-158)

File: .github/workflows/maven-lib-release.yml [hunk 1]
```diff
@@ -1,157 +1,10 @@
...
-    steps:
-      - uses: actions/checkout@v2
-      - name: Update to next minor version
-        run: |
-          /scripts/update-version.sh --java --maven --release pom.xml changelog.md
-        shell: bash
+  run:
+    uses: zepben/.github/.github/workflows/maven-lib-release.yml@main
+    with:
+      private: false
```
Comment (kgreav) [inline .github/workflows/maven-lib-release.yml:9]: We need to make maven-lib-release also support the licencecheck action.

You can use the python-lib-release.yml as a reference in the .github as I implemented it there already.

Once done that you'll need to add `licencecheck: true` here

File: .github/workflows/maven-lib-snapshot.yml [hunk 2]
```diff
@@ -1,52 +1,15 @@
...
-      - uses: actions/checkout@v2
-
-      - name: Update snapshot version
-        run: |
-          /scripts/update-version.sh --java --snapshot --maven pom.xml
-        shell: bash
+  run:
+    uses: zepben/.github/.github/workflows/maven-lib-snapshot.yml@main
+    with:
+      private: false
```
Comment (kgreav) [inline .github/workflows/maven-lib-snapshot.yml:14]: you'll probably also need to add the licencecheck option here.

**Review by kgreav — CHANGES_REQUESTED** [review]
Can you also add the maven-lts-branch.yml and maven-hotfix-branch.yml workflows?

File: .github/workflows/maven-lib-release.yml (hunk 1 above)
Comment (alourie) [reply .github/workflows/maven-lib-release.yml:9]: yup, no worries

**Review by kgreav — APPROVED** [review]

**Pull request merged**